.github/ISSUE_TEMPLATE/DeprecationTemplate.md

```diff
@@ -0,0 +1,18 @@
+name: Deprecation Notice
+about: A notice for one of few API deprecations
+title: 'Deprecation Notice'
+labels: 'kind/deprecation'
+
+---
+
+**The list of APIs to be deprecated**
+List the APIs with their full namespaces and class
```

Comment:
suggestion: It might be enough to say 'List the fully qualified APIs/types', and then have a brief example maybe?